dimod/discrete/discrete_quadratic_model.py

```diff
@@ -793,13 +813,20 @@ def to_numpy_vectors(self):
             :meth:`~DiscreteQuadraticModel.from_numpy_vectors`
 
         """
-        case_starts, linear_biases, quadratic = self._cydqm.to_numpy_vectors()
+        if not return_offset:
+            case_starts, linear_biases, quadratic = self._cydqm.to_numpy_vectors()
```

Comment:
IMO raise a deprecation warning here saying that we will be making `return_offset` default to `True` in the future.

Reply:
Updated. Can we also add @hsadeghidw as a reviewer?

Reply:
I think he can just start reviewing if he wants to